### DISCUSSION TITLE
Creating a Full Site Editing theme in combination with Timber

### DISCUSSION THREAD

**Discussion by kevvr** [timber/timber#2795] [Q&A]
Hey all!

Been using Timber for a year now and looking into setting up a Full Site Editor theme in combination with Timber. However trying to wrap my head around how to set up a custom theme like that is kind of dazzling me a the moment. Would love to hear how you handle these kinds of projects - where to start, how to structure. 

Would you take the base structure of a Full Site Editor theme and work in Timber, or set up a fresh Timber-based theme and then try to implement Full Site Editing capabilities?

Hope to hear from you soon,

Kevin

**Comment by leup** [comment]
I was wondering the same thing. As FSE uses HTML template with comments, I am not sure how it could be handled.
